src/actions/actions.ts

```diff
@@ -3673,6 +3673,11 @@ abstract class MoveByScreenLine extends BaseMovement {
   value: number = 1;
 
   public async execAction(position: Position, vimState: VimState): Promise<Position | IMovement> {
+    if (vimState.currentMode === ModeName.VisualLine) {
```

Comment:
Hmm, actually.

Maybe it is possible? The vscode.selection objects have a start, stop and active attribute. The problem (I think) is that our Range objects only have a start and stop.